.swiftlint.yml

```diff
@@ -0,0 +1,7 @@
+disabled_rules: # rule identifiers to exclude from running
```

Comment:
1. add the documentation, how and when to use it. ( in the section about contribution)
2. add a build phase to run linter automatically with builds

Reply:
3. why configuration is trimmed from https://github.com/realm/SwiftLint#configuration
?

Reply:
1. Ok
2. I've already added a run script to build phases.
3. Isn't it supposed to be trimmed?